Formula/braid.rb

```diff
@@ -0,0 +1,68 @@
+class Braid < Formula
+  desc "Simple tool to help track vendor branches in a Git repository"
+  homepage "https://cristibalan.github.io/braid/"
+  url "https://rubygems.org/gems/braid-1.0.21.gem"
+  sha256 "97300ec6fd9172b25fd05f2c88107e5129a7b85a5f63c73aea40c266b1d80508"
+  head "https://github.com/cristibalan/braid.git"
+
+  resource "arrayfields" do
+    url "https://rubygems.org/gems/arrayfields-4.9.2.gem"
+    sha256 "1593f0bac948e24aa5e5099b7994b0fb5da69b6f29a82804ccf496bc125de4ab"
+  end
+
+  resource "chronic" do
+    url "https://rubygems.org/gems/chronic-0.10.2.gem"
+    sha256 "766f2fcce6ac3cc152249ed0f2b827770d3e517e2e87c5fba7ed74f4889d2dc3"
+  end
+
+  resource "fattr" do
+    url "https://rubygems.org/gems/fattr-2.3.0.gem"
+    sha256 "0430a798270a7097c8c14b56387331808b8d9bb83904ba643b196c895bdf5993"
+  end
+
+  resource "main" do
+    url "https://rubygems.org/gems/main-6.2.2.gem"
+    sha256 "af04ee3eb4b7455eb5ab17e98ab86b0dad8b8420ad3ae605313644a4c6f49675"
+  end
+
+  resource "map" do
+    url "https://rubygems.org/gems/map-6.6.0.gem"
+    sha256 "153a6f384515b14085805f5839d318f9d3c9dab676f341340fa4300150373cbc"
+  end
+
+  resource "open4" do
+    url "https://rubygems.org/gems/open4-1.3.4.gem"
+    sha256 "a1df037310624ecc1ea1d81264b11c83e96d0c3c1c6043108d37d396dcd0f4b1"
+  end
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    resources.each do |r|
+      r.verify_download_integrity(r.fetch)
+      system "gem", "install", r.cached_download, "--ignore-dependencies",
+             "--no-document", "--install-dir", libexec
+    end
+    if head?
+      system "gem", "build", "braid.gemspec"
+      system "gem", "install", Dir.glob("braid-*.gem").last,
+             "--ignore-dependencies", "--no-document", "--install-dir", libexec
+    else
+      system "gem", "install", cached_download,
+             "--ignore-dependencies", "--no-document", "--install-dir", libexec
+    end
+    bin.install libexec/"bin/braid"
+    bin.env_script_all_files(libexec/"bin", :GEM_HOME => ENV["GEM_HOME"])
+  end
+
+  test do
+    system "git", "init"
+    (testpath/"README").write "Testing"
+    (testpath/".gitignore").write "Library"
+    system "git", "add", "README", ".gitignore"
+    system "git", "commit", "-m", "Initial commit"
+    output = shell_output("#{bin}/braid add https://github.com/cristibalan/braid.git")
+    assert_match(/^Braid: Added mirror at '/, output)
+    output = shell_output("#{bin}/braid status")
+    assert_match(/^braid \(/, output)
```

Comment:
Ditto. And this won't need `output` on a separate line to be less than eighty columns wide.